data/strings/strings.txt

```diff
@@ -21084,7 +21086,7 @@
 
   [unable_to_calc_alert_subtitle]
     tags = android,ios
-    en = Unfortunately, we couldn't find a route, probably due to the options you have chosen. Please change the settings and try again.
+    en = Unfortunately a route cannot be found, likely due to the routing options you have enabled. Please disable some and try again.
```

Comment:
Also, it's really hard to say *why*.

```
The app was unable to compute a route.  This could be because of the chosen routing options, because OpenStreetMap data is incorrect or incomplete, or because a route truly does not exist.
```

As an example I just played with OM to compute a route to a restaurant.  I got three different routes for car/bike/walk, with the walking route taking trails in conservation land.  All routes were reasonable.  I then asked for public transit and was told there was no route.   This is also correct as taking a bus does not reduce the walking distance.

Reply:
i don't think it's possible that a route doesn't exist ;)

updated to 
> A route could not be found. This may be caused by your routing options or incomplete OpenStreetMap data. Please change your routing options and retry.

Reply:
It's totally possible.   Try asking for a walking route from someplace in London to the Massachusetts state house.
Or from Juneau, Alaska to Anchorage, except  GraphHopper says to take the ferry, which is not walking.

https://www.openstreetmap.org/directions?engine=graphhopper_foot&route=51.501%2C-0.127%3B42.359%2C-71.064